lib/renaissance/auctions/auctions.ex

```diff
@@ -18,6 +18,13 @@ defmodule Renaissance.Auctions do
     Repo.insert(Auction.changeset(%Auction{}, details))
   end
 
+  def update_description(auction_id, params) do
+    Auction
+    |> Repo.get!(auction_id)
+    |> change(description: params["description"])
```

Comment:
Why do you need to get the auction to update it?

Reply:
This seemed to be the way updates are demoed in the docs, however we may be mistaken.

Reply:
Yeah this is just how Ecto works. Otherwise you'd have to do an "update where" which wouldn't allow you to do validations in your application layer.

Reply:
👍